aries-site/src/examples/templates/dashboards/components/Activity.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import { Anchor, Box, Grid, Text } from 'grommet';
+
+export const Activity = ({
+  icon,
+  title,
+  message,
+  timestamp: timestampProp,
+}) => {
+  const timestamp = new Date(timestampProp);
+
+  return (
+    <Grid columns={['auto', 'flex']} rows={['auto', 'auto']}>
+      <Box justify="center" pad={{ right: 'xsmall' }}>
+        {icon}
+      </Box>
+      <Text weight="bold">{title}</Text>
+      <Box />
+      <Box>
+        <Text>
+          {message} {timestamp.toLocaleDateString()}{' '}
+          {timestamp.toLocaleTimeString()}
+        </Text>
+        <Anchor
+          href="#"
```

Comment:
This is a little confusing because when I close the alert it scrolls me back to the top of the page -- any way to just link back to the example we are on? -- understand this could be difficult if the component is used on various pages, so happy to leave this for a future brainstorm

Reply:
Open to other ideas for how to represent an anchor accurately w/ tab-able focus, etc. Haven't thought of an better approach.

Reply:
That's fair. I can't think of anything at the moment and don't think it's anything that should block this PR.